### PR TITLE
raftstore: tiny update.

### PR DESCRIPTION
**Pull request by siddontang**
@ngaut @BusyJay @tiancaiamao

### CODE REVIEW

File: tests/raftstore/cluster.rs [hunk 1]
```diff
@@ -110,11 +110,14 @@ impl<T: Simulator> Cluster<T> {
     }
 
     pub fn start_with_strategy(&mut self, strategy: Vec<Strategy>) {
-        let mut sim = self.sim.wl();
         for engine in &self.dbs {
-            let node_id = sim.run_node(0, self.cfg.clone(), engine.clone(), strategy.clone());
```
Comment (tiancaiamao) [inline tests/raftstore/cluster.rs]: I just wonder is the old style of writing safe in Rust ?
rest LGTM

Comment (siddontang) [inline tests/raftstore/cluster.rs]: what's the old style?

Comment (tiancaiamao) [inline tests/raftstore/cluster.rs]: let mut sim = self.sim.wl();
then use it in the loop, rather than wl() it every time

Comment (siddontang) [inline tests/raftstore/cluster.rs]: no big different, why I want to move inner is to avoid using `drop sim` before sleep.

Comment (tiancaiamao) [inline tests/raftstore/cluster.rs]: oh, I see.

**Pull request closed without merging**